thinc/backends/mps_ops.py

```diff
@@ -0,0 +1,26 @@
+from typing import TYPE_CHECKING
+import numpy
+
+from .. import registry
+from . import NumpyOps, Ops
+
+if TYPE_CHECKING:
+    # Type checking does not work with dynamic base classes, since MyPy cannot
+    # determine against which base class to check. So, always derive from Ops
+    # during type checking.
+    _Ops = Ops
+else:
+    try:
+        from thinc_apple_ops import AppleOps
+
+        _Ops = AppleOps
+    except ImportError:
+        _Ops = NumpyOps
```

Comment:
Do we want to move this to the `_compat` module (and add an additional `bool` presence flag)?

Reply:
That would be nice! I am a bit worried about introducing import cycles again between thinc and thinc-apple-ops. Let me look if it's possible.

Reply:
Would introduce another cycle. `AppleOps` requires `NumpyOps`. `NumpyOps` requires `util`, `util` requires `compat` and `compat` would import `thinc_apple_ops`.

Reply:
Ah, shame.